tools/SetupFlow/DevHome.SetupFlow/Views/PackageCatalogListView.xaml

```diff
@@ -6,38 +6,103 @@
     xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
     xmlns:viewmodels="using:DevHome.SetupFlow.ViewModels"
     xmlns:views="using:DevHome.SetupFlow.Views"
+    xmlns:converters="using:CommunityToolkit.WinUI.Converters"
+    xmlns:i="using:Microsoft.Xaml.Interactivity"
+    xmlns:ic="using:Microsoft.Xaml.Interactions.Core"
     mc:Ignorable="d">
     <UserControl.Resources>
-        <ResourceDictionary>
-            <!-- Number of items to display per panel -->
-            <x:Int32 x:Key="ItemsPerPanel">6</x:Int32>
-        </ResourceDictionary>
+        <!-- Number of items to display per panel -->
+        <x:Int32 x:Key="ItemsPerPanel">6</x:Int32>
+        <converters:EmptyObjectToObjectConverter x:Key="ViewAllConverter" EmptyValue="Collapsed" NotEmptyValue="Visible" />

```

Comment:
NIT change the key to a more descriptive name.  Something like "EmptyListIsCollapsedConverter"